components/installer/pkg/steps/remove-kyma-components.go

```diff
@@ -2,33 +2,30 @@ package steps
 
 import (
 	"log"
+
+	"github.com/kyma-project/kyma/components/installer/pkg/config"
 )
 
 //RemoveKymaComponents .
-func (steps InstallationSteps) RemoveKymaComponents() {
+func (steps InstallationSteps) RemoveKymaComponents(installationData *config.InstallationData) {
 
 	log.Println("Removing Kyma resources...")
 
-	if err := steps.uninstallReleases(); err != nil {
+	if err := steps.uninstallReleases(installationData); err != nil {
 		log.Println("Error. Some releases may have not been removed.")
 		return
 	}
 
-	if err := steps.removeNamespaces(); err != nil {
-		log.Println("Error. Some namespaces may not have been removed")
-		return
-	}
-
 	log.Println("All Kyma resources have been successfully removed")
 }
 
-func (steps InstallationSteps) uninstallReleases() error {
+func (steps InstallationSteps) uninstallReleases(installationData *config.InstallationData) error {
```

Comment:
Please enclose this function's logic in RemoveKymaComponents() func. There's no need to call another function here.